frontend/src/pages/instance/Settings/Danger.vue

```diff
@@ -228,6 +240,19 @@ export default {
                     this.loading.changingStack = false
                 })
             }
+        },
```

Comment:
clear the query param here too so that if they're refreshing it doesn't keep forcing the scroll/jiggle?

Reply:
good idea